benchmarks/scripts/expval_benchmark.py

```diff
@@ -235,8 +235,37 @@ def simulate_expvals(
 
     else:
         density_matrix = simulate_density_matrix(compiled_circuit)
-        obs_str = "Z" * compiled.num_qubits
```

Comment:
The variable `obs_str` still needs to be set in order to be recorded in the results.

Reply:
Good catch. I wonder why it didn't raise an error when I ran it. Let me check again...

Reply:
Both issues are fixed- 
1. I set `obs_str` to `"H_p"` (problem Hamiltonian) because the sum of Pauli strings is way too long.
2. The missing `obs_str` didn't throw an error because the 'if' statement should have read `if circuit_name == "qaoa_barabasi_albert"` not if circuit_name == "qaoa"`

Reply:
Using `H_p` doesn't seem ideal, but a good compromise. We should record the observable string _somewhere_ for easy reference.

Good find with that bug!